fvm/src/kernel/default.rs

```diff
@@ -371,11 +388,11 @@ where
         }
 
         // Send.
-        let result = self
-            .call_manager
-            .with_transaction(flags.read_only(), |cm| {
-                cm.send::<Self>(from, *recipient, method, params, value, gas_limit)
-            })?;
+        let result = self.call_manager.with_transaction(|cm| {
+            cm.send::<Self>(
```

Comment:
So, technically, instead of passing a "read-only" flag, we could change the kernel's type into a read-only kernel. But it just doesn't seem worth it.